OutOfSchool/OutOfSchool.DataAccess/Models/Workshop.cs

```diff
@@ -72,47 +99,23 @@ public class Workshop : BusinessEntity, IImageDependentEntity<Workshop>, IHasEnt
     [MaxLength(200)]
     public string DisabilityOptionsDesc { get; set; } = string.Empty;
 
-    [Required(ErrorMessage = "Short stay is required")]
-    public bool ShortStay { get; set; } = default;
-
-    [Required(ErrorMessage = "Should be indicated if the Workshop operates with funds from parents or benefactors")]
-    public bool IsSelfFinanced { get; set; } = default;
-
     [Required(ErrorMessage = "Property IsSpecial is required")]
-    public bool IsSpecial { get; set; } = default;
+    public bool IsSpecial { get; set; } = false;
 
-    public uint SpecialNeedsId { get; set; } = 0;
+    public SpecialNeedsType SpecialNeedsType { get; set; } = SpecialNeedsType.None;
```

Comment:
Curios, why this and other enum property is not covered with EnumType attribute, as it was in Dto classes